### ISSUE TITLE
Autocomplete attribute and aria-disabled (73f2c2)

### ISSUE THREAD

**Issue by cblouch** [act-rules/act-rules.github.io#992]
`autocomplete` attribute has valid value
https://act-rules.github.io/rules/73f2c2
has several Applicability checks, one of which is:
The element has a aria-disabled="true" attribute

After some internal debates on implementation of this test it was pointed out that aria-disabled has no actual impact on the behavior of the widget and should not remove it from being tested. In particular was a comment from Bryan Garaventa:

In the case of aria-disabled, the only thing that this attribute does is to set a disabled state flag within the property for that control so it is conveyed for AT users. ATs are not actually supposed to honor anything beyond that. E.G You should still be able to set focus to the form control and interact with in the same way as if it did not include this attribute at all. I believe this is the reasoning why the aria-disabled attribute is being removed from the ‘global attribute’ list within the ARIA spec, and will then be forbidden in accordance with the ARIA-HTML spec. So the disabled attribute alone will be the only valid use of such an attribute on native form fields going forward.

In light of this, it seems like aria-disabled should be removed from the Applicability test exclusion list. As Bryan points out, the control is still going to be there with or without the aria-hidden and so its autocomplete attribute tokens should be checked.

**Comment by jkodu** [comment]
@WilcoFiers can you offer your view on the above.

**Comment by ajanec01** [comment]
Hi @cblouch 

The issue you've raised has been fixed. Feel free to revisit the rule and comment if feel something isn't right.
Here is the link to the rule https://act-rules.github.io/rules/73f2c2
